src/components/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import Icon from './Icon';
+import ProfilePicture from './ProfilePicture';
+import ellipsis from '../assets/ellipsis.svg';
+import Username from './Username';
```

Comment:
Try to have some sort of order to your imports.

at the very least your component imports should be adjacent